content/v1.14/cli/_index.md

```diff
@@ -0,0 +1,60 @@
+---
+weight: 400
+title: Crossplane CLI
+description: "Documentation for the Crossplane command-line interface"
+---
+
+The Crossplane CLI helps simplify some development and administration aspects of
+Crossplane.
+
+The Crossplane CLI includes:
+* tools to build, install, update and push Crossplane Packages
+* standalone Composition Function testing and rendering without the need to access a Kubernetes cluster running Crossplane
+* visual dependency mapping across Compositions, Composite Resources and Managed Resources
+
+## Installing the CLI
+
+The Crossplane CLI is a single standalone binary with no external dependencies.
+
+{{<hint "note" >}}
+Install the Crossplane CLI on a user's computer. 
+
+Most Crossplane CLI commands are independent of Kubernetes and 
+don't require access to a Crossplane pod.
+{{< /hint >}} 
+
+To download the latest version for your CPU architecture with the Crossplane
+install script.
+
+```shell
+curl -sL "https://raw.githubusercontent.com/crossplane/crossplane/master/install.sh" | sh
+```
+
+[The script](https://raw.githubusercontent.com/crossplane/crossplane/master/install.sh)
+detects your CPU architecture and downloads the latest stable release.
+
+Manually download a binary from the Crossplane releases repository at 
```

Comment:
this seems like an alternate option from running the script, but it's reading like the next step in the flow. do you want to make it more clear that it's either run the script OR manually download the binary, not both?

Reply:
Put the manual steps in a collapse section.